rdr_service/resource/generators/pdr_participant.py

```diff
@@ -0,0 +1,223 @@
+#
+# This file is subject to the terms and conditions defined in the
+# file 'LICENSE', which is part of this source code package.
+#
+import os
+
+from rdr_service.dao.resource_dao import ResourceDataDao
+from rdr_service.model.bq_base import BQRecord
+from rdr_service.resource import generators, schemas
+from rdr_service.resource.generators.participant import ParticipantSummaryGenerator
+from rdr_service.resource.schemas.participant import StreetAddressTypeEnum
+
+
+class PDRParticipantSummaryGenerator(generators.BaseGenerator):
+    """
+    Generate a Participant Summary Resource object
+    """
+    ro_dao = None
+    rural_zipcodes = None
+
+    def make_resource(self, p_id, ps_res=None):
+        """
+        Build a Participant Summary BQRecord object for the given participant id.
```

Comment:
Was this comment/description of what is built cut/pasted from the ParticipantSummaryGenerator() class without modification?  Clarify that this is the "de-identified" version of data for PDR?

Reply:
Yes, I'll change the comment.